src/main/proto/envoy/annotations/deprecation.proto

```
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package envoy.annotations;
```

Buf (1.54.0)

Files with package "envoy.annotations" must be within a directory "envoy/annotations" relative to root but were in directory "src/main/proto/envoy/annotations".

(PACKAGE_DIRECTORY_MATCH)